ports/littlegptracker/README.md

```diff
@@ -0,0 +1,32 @@
+## Notes
+Thanks to the [Mdashdotdashn](https://github.com/Mdashdotdashn), [djdiskmachine](https://github.com/djdiskmachine), and other contributors for creating this music tracker and making it available for free.
+
+## Controls
+```
+D-Pad: In screen navigation.
```

Comment:
This is really messy and hard to read. Look up how to use tables in markdown and/or look at how other readmes do it.